iXBRLViewerPlugin/viewer/src/js/inspector.js

```diff
@@ -763,3 +769,23 @@ Inspector.prototype.selectDefaultLanguage = function () {
 Inspector.prototype.setLanguage = function (lang) {
     this._viewerOptions.language = lang;
 }
+
+Inspector.prototype.showValidationReport = function () {
+    const vr = new ValidationReportDialog();
+    vr.displayErrors(this._report.data.validation);
+    vr.show();
+}
+
+Inspector.prototype.setupValidationReportIcon = function () {
+    if (this._report.hasValidationErrors()) {
```

Comment:
To be sure I understand, if either the iXBRL viewer wasn't generated with validations or there aren't any validations, then the iXBRL viewer remains unchanged, right?

Reply:
Yes - exactly so.